pram

```diff
@@ -286,7 +296,16 @@ main() {
 	fi
 
 	if [[ -n ${pr} ]]; then
-		wget -O "${tempdir}/all.patch" "${pr}" || die "Fetching patch failed"
+		if [[ -n ${api} ]]; then
```

Comment:
```suggestion
		if [[ -n ${api} && ${pr} == *://github.com/*/pull/* ]]; then
```

or something like that. We don't want to change `pull/` to `pulls/` for other forges.